backend/fms_core/services/derived_sample.py

```diff
@@ -11,8 +12,8 @@ def inherit_derived_sample(derived_sample_source, new_derived_sample_data):
         new_derived_sample.pk = None
         new_derived_sample.__dict__.update(new_derived_sample_data)
         new_derived_sample.save()
-    except Error as e:
+    except ValidationError as e:
```

Comment:
Is this case wide enough ? I am not sure the update throws only validation errors... This might allow some error to fail to be caught ... I would use `Exception` unless you are sure there is no other errors that can be thrown in there.

Reply:
As far I am concerned the save() function of derived sample returns only ValidationErrors, i.e. `_add_error(errors, field, ValidationError(error))`

Reply:
I would be more preoccupied by `new_derived_sample.__dict__.update(new_derived_sample_data)`